docs/LED_Effect.md

```diff
@@ -326,6 +326,14 @@ the "head" of the comet and the remaining colors are blended into the "tail"
     Palette:          Color of "head" and gradient of "tail"
 Identical settings as Comet, but with multiple lights chasing each other.
 
+#### Cylon (Knight Rider)
```

Comment:
Please remove any mentions of KITT or Knight Rider to avoid any trademark issues.

Reply:
Removed :+1: